libsel4/arch_include/x86/interfaces/sel4arch.xml

```diff
@@ -432,12 +432,12 @@ contain the mapping"/>
             <param dir="in" name="vaddr" type="seL4_Word" description="Virtual address at which to map page."/>
             <param dir="in" name="rights" type="seL4_CapRights_t">
                 <description>
-                    Rights for the mapping. Possible values for this type are given in <autoref label='sec:cap_rights'/>.
+                    Rights for the mapping. <docref>Possible values for this type are given in <autoref label='sec:cap_rights'/>.</docref>
                 </description>
             </param>
             <param dir="in" name="attr" type="seL4_X86_VMAttributes">
                 <description>
-                    VM attributes for the mapping. Possible values for this type are given in <autoref label='ch:vspace'/>.
+                    VM attributes for the mapping.<docref> Possible values for this type are given in <autoref label='ch:vspace'/>.</docref>
```

Comment:
(or if the whitespace creates strange output, we should change the other one a few lines above to match up with this one)
```suggestion
                    VM attributes for the mapping. <docref>Possible values for this type are given in <autoref label='ch:vspace'/>.</docref>
```